app/src/main/java/com/github/onlynotesswent/ui/overview/EditNote.kt

```diff
@@ -207,7 +209,10 @@ fun EditNoteScreen(
                   onClick = {
                     updatedComments =
                         Note.CommentCollection.addComment(
-                            userViewModel.currentUser.value?.uid ?: "1", "", updatedComments)
```

Comment:
This implementation is weird. See larger comment on a bigger refactor of CommentCollection class, which will require changes here.

Reply:
To be discussed later